### DISCUSSION TITLE
having trouble with installing

### DISCUSSION THREAD

**Discussion by HeyySuraj** [spiral-project/ihatemoney#824] [Q&A] [answered]
I really need to contribute this project but I unable to install this project locally in windows 10 although Im trying with Docker Tool ..plz help me asap

**Comment by Glandos** [comment, marked as answer]
Hi,

In order for other people help you please provide more details of what is going wrong. I am not a user of Windows, but be sure you fulfill all requirements from https://docs.docker.com/desktop/windows/wsl/

Then don't hesitate to share any log files, screenshots,…

**Comment by HeyySuraj** [comment]
https://ibb.co/4YQdgcF 

I go through the path your suggest in but stuck with this problem (provide screenshot in the link)

**Reply by HeyySuraj** [reply]
error during connect: Post "https://192.168.99.100:2376/v1.41/volumes/create": dial tcp 192.168.99.100:2376: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 


as ss in not clear I send the error
